ui/web/src/main/java/com/ttbmp/cinehub/ui/web/buyticket/ChooseSeatsViewController.java

```diff
@@ -0,0 +1,99 @@
+package com.ttbmp.cinehub.ui.web.buyticket;
+
+import com.ttbmp.cinehub.app.dto.ProjectionDto;
+import com.ttbmp.cinehub.app.dto.TicketDto;
+import com.ttbmp.cinehub.app.usecase.buyticket.BuyTicketUseCase;
+import com.ttbmp.cinehub.app.usecase.buyticket.Handler;
+import com.ttbmp.cinehub.app.usecase.buyticket.request.GetNumberOfSeatsRequest;
+import com.ttbmp.cinehub.app.usecase.buyticket.request.GetProjectionRequest;
+import com.ttbmp.cinehub.ui.web.domain.Projection;
+import com.ttbmp.cinehub.ui.web.domain.Seat;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Palmieri Ivan
+ */
+@Controller
+public class ChooseSeatsViewController {
```

Comment:
The controller should send request to the server, the presenter should handle the server response and update the model